corgidrp/l2b_to_l3.py

```diff
@@ -19,13 +20,31 @@ def divide_by_exptime(input_dataset):
     
     Divide the data by the exposure time to get the units in electrons/s
 
-    TODO: Make sure to update the headers to reflect the change in units
-
     Args:
         input_dataset (corgidrp.data.Dataset): a dataset of Images (L2b-level)
 
     Returns:
         corgidrp.data.Dataset: a version of the input dataset with the data in units of electrons/s
     """
 
-    return None
+    data = input_dataset.copy()
+
+    all_data_new = np.zeros(data.all_data.shape)
+    all_err_new = np.zeros(data.all_err.shape)
+
+    for i in range(len(data.frames)):
+        exposure_time = float(data.frames[i].ext_hdr['EXPTIME'])
+
+        data.frames[i].data = data.frames[i].data / exposure_time
+        data.frames[i].err = data.frames[i].err / exposure_time
```

Comment:
@maxwellmb, I think that is not necessary in this case since exposure_time has no uncertainty, so a simple division of the errors should be ok. Dividing the flat field is something different there you need a proper error propagation.

Reply:
I think we'll still want to use the `rescale_errors` function here, it's just that it won't have a pixel-dependent scaling as we do in the flats. I do agree that the flat is different though, since there are kind of two steps there with the additive component.